commit 83b0c36b8a136042aa9209fc9ca2fab31d5fbe7c
Author: Sai Prakash Ranjan <quic_saipraka@quicinc.com>
Date:   Fri Jan 17 01:33:53 2020 +0530

    arm64: Add KRYO{3,4}XX CPU cores to spectre-v2 safe list
    
    The "silver" KRYO3XX and KRYO4XX CPU cores are not affected by Spectre
    variant 2. Add them to spectre_v2 safe list to correct the spurious
    ARM_SMCCC_ARCH_WORKAROUND_1 warning and vulnerability status reported
    under sysfs.
    
    Reviewed-by: Stephen Boyd <swboyd@chromium.org>
    Tested-by: Stephen Boyd <swboyd@chromium.org>
    Signed-off-by: Sai Prakash Ranjan <saiprakash.ranjan@codeaurora.org>
    [will: tweaked commit message to remove stale mention of "gold" cores]
    Signed-off-by: Will Deacon <will@kernel.org>

diff --git a/arch/arm64/kernel/cpu_errata.c b/arch/arm64/kernel/cpu_errata.c
index 85f4bec22f6d..d661c7d0e6e2 100644
--- a/arch/arm64/kernel/cpu_errata.c
+++ b/arch/arm64/kernel/cpu_errata.c
@@ -548,6 +548,8 @@ static const struct midr_range spectre_v2_safe_list[] = {
 	MIDR_ALL_VERSIONS(MIDR_CORTEX_A55),
 	MIDR_ALL_VERSIONS(MIDR_BRAHMA_B53),
 	MIDR_ALL_VERSIONS(MIDR_HISI_TSV110),
+	MIDR_ALL_VERSIONS(MIDR_QCOM_KRYO_3XX_SILVER),
+	MIDR_ALL_VERSIONS(MIDR_QCOM_KRYO_4XX_SILVER),
 	{ /* sentinel */ }
 };